storage/src/database/tezedge_database.rs

```diff
@@ -0,0 +1,355 @@
+use crate::database::backend::{
+    BackendIterator, BackendIteratorMode, TezedgeDatabaseBackendStore,
+    TezedgeDatabaseBackendStoreIterator,
+};
+use crate::database::error::Error;
+use crate::database::sled_backend::SledDBBackend;
+use crate::persistent::{Decoder, Encoder, KeyValueSchema, SchemaError};
+use crate::{IteratorMode, OperationsStorage, OperationsMetaStorage, BlockMetaStorage, SystemStorage, PredecessorStorage, ChainMetaStorage};
+use std::marker::PhantomData;
+use std::sync::Arc;
+use std::sync::atomic::{AtomicU64, Ordering};
+use serde::{Serialize, Deserialize};
+use im::HashMap;
+use crate::block_storage::{BlockPrimaryIndex, BlockByContextHashIndex, BlockByLevelIndex};
+use crate::persistent::sequence::Sequences;
+use crate::database::notus_backend::NotusDBBackend;
+
+pub trait KVStoreKeyValueSchema: KeyValueSchema {
+    fn column_name() -> &'static str;
+}
+
+/// Custom trait to unify any kv-store schema access
+pub trait KVStore<S: KeyValueSchema> {
```

Comment:
@mambisi 
I would name this `KvStore` insteaed of `KVStore`

Reply:
Ok